spec/environment/environment.rb

```diff
@@ -25,6 +26,8 @@ def load(env = ENV['GD_ENV'] || 'testing')
         # VCR is enabled by default - set VCR_ON=false to disable
         vcr_on = ENV['VCR_ON'].nil? || ENV['VCR_ON'].downcase == 'true'
         GoodData::Environment.set_const(:VCR_ON, vcr_on)
+        suffix = SecureRandom.urlsafe_base64(5)
```

Comment:
I remember once we had a problem with randomly generated names. Some API endpoint wouldn't accept names with certain special characters (underscores?). It might not be a problem here but I feel I should warn you. ⚠️